### PR TITLE
WMS 11184: Review of Extended Lab to be used for OCW Take 4

### PR DESCRIPTION
**Pull request by yiby4n**
# Add here your WMS ID 11184

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

### CODE REVIEW

File: oci-stack-monitoring-fundamentals/access-env/access-env.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Access Stack Monitoring Workshop environment
+# Accessing the Workshop environment
```
Comment (anooshapilli) [inline oci-stack-monitoring-fundamentals/access-env/access-env.md:1]: @yiby4n - Please update the title to use Access instead of Accessing.

**Review by anooshapilli — COMMENTED** [review]
@yiby4n - 
- Could you rename the image file name to remove ~ and delete the videos from the repo? They take space from the repo. Please upload to Videohub and link the video to the lab.
image
- We recommend using imperative titles instead of a gerund. For example, 'Start' instead of 'Starting'. Please update the lab titles in the markdown files.

File: oci-stack-monitoring-fundamentals/access-env/access-env.md (hunk 1 above)
Comment (yiby4n) [reply oci-stack-monitoring-fundamentals/access-env/access-env.md]: Done. Thank you

File: oci-stack-monitoring-fundamentals/together/together.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# Applying what you have learned
```
Comment (anooshapilli) [inline oci-stack-monitoring-fundamentals/together/together.md]: Update the title to "Apply" in both the manifest.json and markdown files

File: oci-stack-monitoring-fundamentals/workshops/freetier/manifest.json [hunk 3]
```diff
@@ -28,6 +27,21 @@
             "description": "Get to know Stack Monitoring's Resource Homepages",
             "filename": "../../homepages/homepages.md"
         },
+        {
+            "title": "Lab 4: Resource Discovery",
+            "description": "Review Stack Monitoring's Resource Discovery Process",
+            "filename": "../../discovery/discovery.md"
+        },
+        {
+            "title": "Lab 5: Viewing and Creating Alarms",
```
Comment (anooshapilli) [inline oci-stack-monitoring-fundamentals/workshops/freetier/manifest.json]: @yiby4n - Update the Lab 5 title to View and Create Alarms in both the manifest.json and markdown files

File: oci-stack-monitoring-fundamentals/workshops/freetier/manifest.json [hunk 4]
```diff
@@ -28,6 +27,21 @@
...
+            "description": "Review Stack Monitoring's Resource Discovery Process",
+            "filename": "../../alarms/alarms.md"
+        },
+        {
+            "title": "Lab 6: Putting it all together",
```
Comment (anooshapilli) [inline oci-stack-monitoring-fundamentals/workshops/freetier/manifest.json]: Update Lab 6 title to Put in both the manifest.json and markdown files

**Review by anooshapilli — APPROVED** [review]

**Pull request merged**